inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -398,43 +398,29 @@ CNNNetworkNGraphImpl::reshape(const std::map<std::string, std::vector<size_t>>&
     return OK;
 }
 
-StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath, const std::string& binPath,
-                                           ResponseDesc* resp) const noexcept {
-    auto network = cnnNetwork;
-    if (!network) {
-        // TODO: once Serialization::SerializeV10 supports true IR v10
-        // remove this conversion and WA for execution graph
-        try {
-            bool isExecutionGraph = true;
-            for (const auto & op : _ngraph_function->get_ops()) {
-                auto & rtInfo = op->get_rt_info();
-                if (rtInfo.find(ExecGraphInfoSerialization::PERF_COUNTER) == rtInfo.end()) {
-                    isExecutionGraph = false;
-                    break;
-                }
-            }
-            if (isExecutionGraph) {
-                Serialization::SerializeV10(xmlPath, binPath, (InferenceEngine::ICNNNetwork&)*this);
-                return OK;
-            }
-
-#ifdef ENABLE_V7_SERIALIZE
-            network = std::make_shared<details::CNNNetworkImpl>(*this);
-#endif
-        } catch (const InferenceEngineException& e) {
-            return DescriptionBuffer(GENERAL_ERROR, resp) << e.what();
-        } catch (const std::exception& e) {
-            return DescriptionBuffer(UNEXPECTED, resp) << e.what();
-        } catch (...) {
-            return DescriptionBuffer(UNEXPECTED, resp);
-        }
-    }
-
+StatusCode CNNNetworkNGraphImpl::serialize(const std::string &xmlPath,
+                                           const std::string &binPath,
+                                           ResponseDesc *resp) const noexcept {
+  try {
```

Comment:
May I ask you to use 4 spaces instead of 2 for indentation?

Reply:
I tried to align to IE style rules in this [commit](https://github.com/openvinotoolkit/openvino/pull/2579/commits/d31bd0a3d7e2ccf7bf7ad1d4d8f424d31151eb61)

I used .clang-format as bellow - we can enhance it if needed. It is based on [this one](https://gitlab-icv.inn.intel.com/inference-engine/dldt/blob/0b740dffb82d63a323c194721380efe6248310d9/inference-engine/.clang-format), but with removed column limit (it looked like other code didn't adhere to 160 limit) and few other unsupported by my clang-format version. Either way it would be useful if we maintain .clang-format for IE even if we can use it only unofficially.

```
jdanieck@ubuntu-ngraph:~/openvino/inference-engine$ clang-format --version
clang-format version 3.9.1-19ubuntu1 (tags/RELEASE_391/rc2)
jdanieck@ubuntu-ngraph:~/openvino/inference-engine$ cat .clang-format 
BasedOnStyle: Google
IndentWidth: 4
UseTab: Never
---
Language: Cpp
Standard: Cpp11

AccessModifierOffset: -4
AllowShortFunctionsOnASingleLine: Empty
AlwaysBreakBeforeMultilineStrings: false
DerivePointerAlignment: false
IndentCaseLabels: false
---
```